librepo/downloader.c

```diff
@@ -1919,6 +1919,9 @@ lr_perform(LrDownload *dd, GError **err)
             else
                 timeout.tv_usec = (curl_timeout % 1000) * 1000;
         }
+        else if (!still_running)
+            // do not sleep for 1s if no more transfers are running
```

Comment:
Minor style thing, can add braces?  The C parser in my brain doesn't like what's basically a multi-line statement even though it's a comment.